examples/worker-react/web/src/main/App.tsx

```diff
@@ -1,5 +1,6 @@
-import React, { useMemo, useState } from 'react'
+import { useMemo, useState } from 'react'
```

Comment:
Just removing React import as it shows as unused (and isn't necessary as of React >17)